docs/settings.md

```diff
@@ -56,27 +56,28 @@ The following tables list available configuration settings with their default va
+| `trivy.imageRef`                      | `docker.io/aquasec/trivy:0.16.0`                                                  | Trivy image reference |
+| `trivy.mode`                          | `Standalone`                                                                      | Trivy client mode. Either `Standalone` or `ClientServer`. Depending on the active mode other settings might be applicable or required. |
+| `trivy.serverURL`                     | N/A                                                                               | The endpoint URL of the Trivy server. Required in `ClientServer` mode. |
+| `trivy.serverTokenHeader`             | `Trivy-Token`                                                                     | The name of the HTTP header to send the authentication token to Trivy server. Only application in `ClientServer` mode when `trivy.serverToken` is specified. |
+| `trivy.insecureRegistry.<id>`         | N/A                                                                               | The registry to which insecure connections are allowed. There can be multiple registries with different registry `<id>`. |
+| `aqua.imageRef`                       | `docker.io/aquasec/scanner:5.3`                                                   | Aqua scanner image reference. The tag determines the version of the `scanner` binary executable and it must be compatible with version of Aqua console. |
+| `aqua.serverURL`                      | N/A                                                                               | The endpoint URL of Aqua management console |
+| `kube-bench.imageRef`                 | `docker.io/aquasec/kube-bench:0.5.0`                                              | kube-bench image reference |
+| `kube-hunter.imageRef`                | `docker.io/aquasec/kube-hunter:0.4.1`                                             | kube-hunter image reference |
+| `kube-hunter.quick`                   | `"false"`                                                                         | Whether to use kube-hunter's "quick" scanning mode (subnet 24). Set to `"true"` to enable. |
+| `polaris.imageRef`                    | `quay.io/fairwinds/polaris:3.2`                                                   | Polaris image reference |
+| `polaris.config.yaml`                 | [Check the default value here][default-polaris-config]                            | Polaris configuration file |
+| `conftest.imageRef`                   | `docker.io/openpolicyagent/conftest:v0.25.0`                                      | Conftest image reference |
+| `vulnerabilityReports.podTolerations` | `'[{"key":"key1", "operator":"Equal", "value":"value1", "effect":"NoSchedule"}]'` | JSON representation of the [tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration) to be applied to the vulnerability scanner pods so that they can run on nodes with matching taints. |
```

Comment:
Sure that makes sense.  I can do that shortly.

Reply:
Actually, as I was making this change, it occurred to me that since only vuln scans support tolerations at this point, it might be  best to keep the name of the param specific to vuln scans.  Then when the param is generalized it can be renamed to reflect that.  For now I think it would be a source of confusion to name it as if it would apply to all scans when it doesn't.  Do you still think we should rename it?

Reply:
I'd rather avoid renaming config params and use a proper / generic name from the get go and apply tolerations config to other scanners before we release new version.

Deprecating config params and keeping backward compatibility is expensive. How would you migrate config objects? For example:
1. You install the Operator with vulnerabilityReport.tolerations param
2. Then we release new version of the Operator with scanJob.tolerations
3. You upgrade the Operator without modifying the starboard ConfigMap
4. You end up with invalid tolerations config (= no config) because we renamed the config key

Reply:
That's a good point.  So yeah, I'll rename the param and make note in the settings doc that it's only supported on vuln scans for the time being.

Reply:
This is done.